kratos/utilities/compare_elements_and_conditions_utility.cpp

```diff
@@ -0,0 +1,78 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main author: Carlos Roig
+//               Vicente Mataix Ferrandiz
+//
+
+
+// System includes
+#include <typeinfo>
+
+// External includes
+
+// Project includes
+#include "includes/kratos_components.h"
+#include "utilities/compare_elements_and_conditions_utility.h"
+
+namespace Kratos
+{
+void CompareElementsAndConditionsUtility::GetRegisteredName(const Geometry<Node<3>>& rGeometry, std::string& rName) {
```

Comment:
```suggestion
void CompareElementsAndConditionsUtility::GetRegisteredName(
    const Geometry<Node<3>>& rGeometry, 
    std::string& rName)
{
```
(minor style guide suggestion)